recipes/python/retail/retail/scoringdataloader.py

```diff
@@ -37,29 +38,35 @@ def load(configProperties):
     if (timeframe is not None):
         date_before = datetime.utcnow().date()
         date_after = date_before - timedelta(minutes=int(timeframe))
-        df = prodreader.load(data_set_id=scoring_data_set_id, ims_org=configProperties['ML_FRAMEWORK_IMS_TENANT_ID'], date_after=date_after, date_before=date_before)
+        dataframe = prodreader.load(data_set_id=scoring_data_set_id, ims_org=configProperties['ML_FRAMEWORK_IMS_TENANT_ID'], date_after=date_after, date_before=date_before)
     else:
-        df = prodreader.load(data_set_id=scoring_data_set_id, ims_org=configProperties['ML_FRAMEWORK_IMS_TENANT_ID'])
+        dataframe = prodreader.load(data_set_id=scoring_data_set_id, ims_org=configProperties['ML_FRAMEWORK_IMS_TENANT_ID'])
+
+
+    if '_id' in dataframe.columns:
+        util = Utils()
+        dataframe = util.renameColumns(configProperties, dataframe)
+
 
     #########################################
     # Data Preparation/Feature Engineering
     #########################################
```

Comment:
@rjrussell77  I have renamed df to dataframe to have the same names in all classes.